WooCommerce/Classes/ViewModels/ProductDetailsCellViewModel.swift

```diff
@@ -1,15 +1,37 @@
 import Foundation
 import Yosemite
 
+protocol AnyAttributeDisplayable {
+    var name: String { get set }
+    var value: String { get set }
+    var nameOrValue: String { get }
+}
+
+struct VariationAttributeViewModel: AnyAttributeDisplayable {
+
+    var name: String = ""
+
+    var value: String = ""
```

Comment:
Do you have any reason to have these properties as `var`s. If they are not going to be mutated I think having them as `let`s express the intention better to the reader!

Reply:
Fixed in d68c3d3!